felkerPgh.md

```diff
@@ -0,0 +1,18 @@
+# Pittsburgh PA
+
+##Places to eat
```

Comment:
@felkerdj2 Make sure to have a space between `##` and the text so the header works properly. 👍